packages/client-sdk-nodejs/package.json

```diff
@@ -0,0 +1,59 @@
+{
+  "name": "@gomomento/sdk",
+  "version": "0.0.1",
+  "description": "Client SDK for Momento services",
+  "main": "dist/src/index.js",
+  "files": [
+    "dist/src",
+    "dist/package.json"
+  ],
+  "types": "dist/src/index.d.ts",
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/momentohq/client-sdk-nodejs"
+  },
+  "scripts": {
+    "prebuild": "eslint . --ext .ts",
+    "test": "jest",
+    "unit-test": "jest unit",
+    "integration-test": "jest integration",
+    "lint": "eslint . --ext .ts",
+    "format": "eslint . --ext .ts --fix",
+    "watch": "tsc -w",
+    "build": "rm -rf dist && tsc"
+  },
+  "author": "",
+  "license": "Apache-2.0",
+  "devDependencies": {
+    "@types/google-protobuf": "^3.15.5",
+    "@types/jest": "^27.0.2",
+    "@types/node": "14.18.3",
+    "@types/uuid": "^8.3.1",
+    "@typescript-eslint/eslint-plugin": "^5.0.0",
+    "@typescript-eslint/parser": "^5.0.0",
+    "eslint": "^7.32.0",
+    "eslint-config-prettier": "^8.3.0",
+    "eslint-config-standard": "^16.0.3",
+    "eslint-plugin-import": "^2.25.1",
+    "eslint-plugin-node": "^11.1.0",
+    "eslint-plugin-prettier": "^4.0.0",
+    "eslint-plugin-promise": "^5.1.0",
+    "jest": "^29",
+    "jest-extended": "^3.2.3",
+    "prettier": "^2.4.1",
+    "ts-jest": "^29",
+    "ts-node": "^10.3.0",
+    "typescript": "^4.4.3",
+    "uuid": "8.3.2"
+  },
+  "dependencies": {
+    "@gomomento/generated-types": "0.51.1",
+    "@grpc/grpc-js": "1.7.3",
+    "google-protobuf": "^3.21.2",
+    "jwt-decode": "3.1.2",
+    "@gomomento/common": "file:../common"
```

Comment:
Do you know how this will work with the packaging?  Or maybe I'll find that out when I look at your build script.